backend/src/boundary_stats.rs

```diff
@@ -210,5 +224,22 @@ mod tests {
         assert_relative_eq!(boundary_stats.population, 60. + 6000.);
         assert_relative_eq!(boundary_stats.simd, 7.2 + 24.);
         assert_eq!(boundary_stats.number_stats19_collisions, 0);
+        assert_eq!(boundary_stats.pois, 0);
     }
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct POI {
+    pub point: Point,
+    pub kind: POIKind,
+    pub name: Option<String>,
+}
+
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub enum POIKind {
```

Comment:
I'll work with Cici to refine these categories. We may want. Maybe GPs and hospitals should be combined. The only purpose of distinguishing is for the layer, not the metric.